SmartDeviceLink/SDLIAPTransport.m

```diff
@@ -32,8 +33,9 @@ @interface SDLIAPTransport () {
 }
 
 @property (assign) int retryCounter;
-@property (assign) BOOL sessionSetupInProgress;
 @property (strong) SDLTimer *protocolIndexTimer;
+@property (nonatomic, assign) UIBackgroundTaskIdentifier backgroundTaskId;
+@property (nonatomic, assign, setter=setSessionSetupInProgress:) BOOL sessionSetupInProgress;
```

Comment:
`, setter=setSessionSetupInProgress:` is superfluous, it doesn't add or change anything.